packages/dataviews/src/dataviews-layouts/list/style.scss

```diff
@@ -34,6 +34,7 @@ ul.dataviews-view-list {
 				flex-basis: min-content;
 				overflow: unset;
 				margin-inline: $grid-unit-10 0;
+				padding-right: $grid-unit-05;
```

Comment:
Super nit as from testing RTL, this gets changed to `padding-left`. Though using a logical property i.e. `padding-inline-end` would be nice, me thinks.